templates/api/knowledge-panels/root.tt.json

```diff
@@ -14,6 +14,12 @@
                 "panel_id": "environment_card",
             },
         },
+        {
+            "element_type": "panel",
+            "panel_element": {
+                "panel_id": "report_problem_card",
+            },
+        },        
```

Comment:
minor: You put a if in the knowledge panel construction method, but not here !

Reply:
the if is for the signalconso panel, the report_problem_card is always there.